class.json-api-endpoints.php

```diff
@@ -1053,11 +1053,13 @@ function user_can_view_post( $post_id ) {
 	 * Returns author object.
 	 *
 	 * @param $author user ID, user row, WP_User object, comment row, post row
-	 * @param $show_email output the author's email address?
+	 * @param $show_email_and_ip output the author's email address and IP address?
 	 *
-	 * @return (object)
+	 * @return object
 	 */
-	function get_author( $author, $show_email = false ) {
+	function get_author( $author, $show_email_and_ip = false ) {
+		$ip_address = $author->comment_author_IP;
```

Comment:
If this is not set, isn't enough the cast to `(string)` we do later to return an empty string?